src/buffer_manager.cc

```diff
@@ -78,7 +78,8 @@ void BufferManager::backup_modified_buffers()
 {
     for (auto& buf : m_buffers)
     {
-        if ((buf->flags() & Buffer::Flags::File) and buf->is_modified())
+        if ((buf->flags() & Buffer::Flags::File) and buf->is_modified()
+            and !(buf->flags() & Buffer::Flags::ReadOnly))
```

Comment:
involuntary deviation from the coding standard, we should fix them when we see them.